node/sc/subbridge.go

```diff
@@ -210,12 +209,44 @@ func NewSubBridge(ctx *node.ServiceContext, config *SCConfig) (*SubBridge, error
 	return sc, nil
 }
 
+func (sb *SubBridge) SetRPCConn(conn net.Conn) {
+	sb.rpcConn = conn
+	go func() {
+		for {
+			data := make([]byte, rpcBufferSize)
+			rlen, err := conn.Read(data)
+			if err != nil || rlen <= 0 {
+				//logger.Error("failed to read from RPC server pipe", "err", err, "rlen", rlen)
+				time.Sleep(100 * time.Millisecond)
+				continue
+			}
```

Comment:
Could you let me know why this delay and continue routines are needed?

Reply:
If EN goes down, EOF error will occur continuously. Because of this, we have inserted a sleep so that the server CPU usage is not affected much.